dictcc.el

```diff
@@ -61,6 +61,37 @@
   :type 'string
   :group 'dictcc)
 
+(defcustom dictcc-languages-alist '(("English" . "en")
+                                           ("German" . "de")
+                                           ("Swedish" . "sv")     
+                                           ("Icelandic" . "is")     
+                                           ("Russian" . "u")     
+                                           ("Romanian" . "ro")     
+                                           ("Italian" . "it")     
+                                           ("French" . "fr")     
+                                           ("Portuguese" . "pt")     
+                                           ("Hungarian" . "hu")     
+                                           ("Dutch" . "nl")     
+                                           ("Slovak" . "sk")     
+                                           ("Latin" . "la")     
+                                           ("Finnish" . "fi")     
+                                           ("Spanish" . "es")     
+                                           ("Bulgarian" . "bg")     
+                                           ("Croation" . "hr")     
+                                           ("Norwegian" . "no")     
+                                           ("Czech" . "cs")     
+                                           ("Danish" . "da")     
+                                           ("Turkish" . "tr")     
+                                           ("Polish" . "pl")     
+                                           ("Serbian" . "sr")     
+                                           ("Greek" . "el")     
+                                           ("Esperanto" . "eo")     
+                                           ("Bosnian" . "bs")     
+                                           ("Albanian" . "sq"))
```

Comment:
Now the tabs are gone but something is still off. Could you re-format the buffer one last time?